bciers/libs/utils/src/customTransformErrors.test.ts

```diff
@@ -130,7 +268,35 @@ describe("customTransformErrors", () => {
     );
 
     expect(transformedErrors[0].message).not.toBe(originalErrorMessage);
-    expect(transformedErrors[0].message).toBe("Required field");
+    expect(transformedErrors[0].message).toBe("Legal name is required");
+  });
+
+  it("returns the transformed error message for a nested required field", () => {
+    const sectionedLatitude = [
+      {
+        name: "required",
+        property:
+          ".facility_information_array.0.section1.latitude_of_largest_emissions",
+        message: "must have required property 'email'",
```

Comment:
why is the message for 'email' when it's the lat/long field?

Reply:
because I am silly